### PR TITLE
changing sha for config submodule, update README, update command datadir copy

### PR DESCRIPTION
**Pull request by jeanmi151**

### CODE REVIEW

**Comment by edevosc2c** [comment]
This is for switching to the new docker-master branch?

**Comment by jeanmi151** [comment]
> This is for switching to the new docker-master branch?

yes

File: README.md [hunk 1]
```diff
@@ -104,14 +104,34 @@ Images and configuration are updated regularly.
...
 To upgrade, we recommend you to:
  * update the configuration with `git submodule update`
- * update the software with `docker-compose pull`
+ * update the software with `docker compose pull`
 
 
 ## Customising
 
-Adjust the configuration in the `config` folder according to your needs.
+This docker composition support environnement variable, if you need to customize something it might be in the different env var files.
```
Comment (pmauduit) [inline README.md]: - support*s*
- variable*s* (I'm pretty sure there are more than one :D)

File: README.md [hunk 2]
```diff
@@ -104,14 +104,34 @@ Images and configuration are updated regularly.
...
+
+Here is the list this files:
```
Comment (pmauduit) [inline README.md]: "of these files" ?

File: README.md [hunk 3]
```diff
@@ -104,14 +104,34 @@ Images and configuration are updated regularly.
...
+- [.envs-common](.envs-common) 
+- [.envs-database-datafeeder](.envs-database-datafeeder)
+- [.envs-database-georchestra](.envs-database-georchestra)
+- [.envs-hosts](.envs-hosts)
+- [.envs-ldap](.envs-ldap)
+
+If you add variable be careful because it might be added into the wrong/unwanted container.
+
+You can also add environnement variable directly into the docker-compose.yaml if needed.
+
```
Comment (pmauduit) [inline README.md]: plural variables

File: README.md [hunk 4]
```diff
@@ -104,14 +104,34 @@ Images and configuration are updated regularly.
...
+To check which container is including which envs file you can look at the docker-compose* files and search for the .envs-* filename wanted.
+
+If you don't find the value in it, there is still a lot to
+adjust the configuration in the `config` folder according to your needs.
 Reading the [quick configuration guide](https://github.com/georchestra/datadir/blob/docker-master/README.md) might help !
 
+Also in production environnement don't forget to change le file contains into the [secret/](secrets/) folder as they are default password.
```
Comment (pmauduit) [inline README.md]: "to change the file into the secret/ folder"

File: README.md [hunk 5]
```diff
@@ -104,14 +104,34 @@ Images and configuration are updated regularly.
...
+
+For [geoserver_privileged_user_passwd.txt](secrets/geoserver_privileged_user_passwd.txt) it need to be the same that in the datadir : https://github.com/georchestra/datadir#3-steps-editing
```
Comment (pmauduit) [inline README.md]: it needs

File: docker-compose.yml [hunk 6]
```diff
@@ -24,7 +24,7 @@ secrets:
 services:
   copy-datadir:
     image: alpine
-    command: sh -c "rm -r /etc/georchestra/* && cp -r -f -v /mnt/datadir/* /etc/georchestra/ && chmod 777 -R -v /etc/georchestra/" # "sleep 6000"
+    command: sh -c "rm -r /etc/georchestra/* ; cp -r -f -v /mnt/datadir/* /etc/georchestra/ ; chmod 777 -R -v /etc/georchestra/" # "sleep 6000"
```
Comment (pmauduit) [inline docker-compose.yml:27]: :+1:

File: README.md [hunk 7]
```diff
@@ -75,7 +75,7 @@ The current FQDN `georchestra-127-0-1-1.traefik.me` resolves to 127.0.1.1, thank
 To change it:
  * Rename the traefik service in the `docker-compose.override.yml` file to match the new domain
  * Modify the three `traefik.http.routers.*.rule` in the `docker-compose.override.yml` file
- * Update the datadir in the config folder (hint: grep for `georchestra-127-0-1-1.traefik.me`)
+ * Update the variable FQDN in [.envs-common](.envs-common) file (hint: grep for `georchestra-127-0-1-1.traefik.me`)
```
Comment (fvanderbiest) [inline README.md]: "Update the FQDN variable"

File: README.md [hunk 8]
```diff
@@ -104,14 +104,34 @@ Images and configuration are updated regularly.
...
 To upgrade, we recommend you to:
  * update the configuration with `git submodule update`
- * update the software with `docker-compose pull`
+ * update the software with `docker compose pull`
 
 
 ## Customising
 
-Adjust the configuration in the `config` folder according to your needs.
+This docker composition supports environnement variables, if you need to customize something it might be in the different env var files.
```
Comment (fvanderbiest) [inline README.md]: environment variables

File: README.md [hunk 9]
```diff
@@ -104,14 +104,34 @@ Images and configuration are updated regularly.
...
+Here is the list of these files:
+- [.envs-common](.envs-common) 
+- [.envs-database-datafeeder](.envs-database-datafeeder)
+- [.envs-database-georchestra](.envs-database-georchestra)
+- [.envs-hosts](.envs-hosts)
+- [.envs-ldap](.envs-ldap)
+
+If you add variable be careful because it might be added into the wrong/unwanted container.
+
+You can also add environnement variables directly into the docker-compose.yaml if needed.
```
Comment (fvanderbiest) [inline README.md]: there are other "environnement" occurences too...

File: README.md [hunk 10]
```diff
@@ -104,14 +104,34 @@ Images and configuration are updated regularly.
...
+This docker composition supports environment variables, if you need to customize something it might be in the different environment variables files.
+
+Here is the list of these files:
+- [.envs-common](.envs-common) 
+- [.envs-database-datafeeder](.envs-database-datafeeder)
+- [.envs-database-georchestra](.envs-database-georchestra)
+- [.envs-hosts](.envs-hosts)
+- [.envs-ldap](.envs-ldap)
+
+If you add variable be careful because it might be added into the wrong/unwanted container.
```
Comment (pmauduit) [inline README.md]: variables,

**Review by pmauduit — APPROVED** [review]

**Pull request merged**